spec/jobs/facilities/mental_health_reload_job_spec.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Facilities::MentalHealthReloadJob, type: :job do
+  class FakeMentalHealthClient
+    def download
+      [
+        {
+          'StationNumber' => '101A', 'MHPhone' => '407-123-1234', 'Extension' => '0001', 'Modified' => '2019-09-06T13:00:00.000'
+        },
+        {
+          'StationNumber' => '202A', 'MHPhone' => '321-123-1234', 'Extension' => '0002', 'Modified' => '2019-09-06T13:00:00.000'
+        }
+      ]
+    end
+  end
+
+  before(:each) do
+    FacilityMentalHealth.keys.map { |k| FacilityMentalHealth.delete(k) }
```

Comment:
This cleanup is handled for you, isn't it?

Reply:
Is it? I wasn't sure, which is why I added this.